src/clusterfuzz/_internal/google_cloud_utils/storage.py

```diff
@@ -22,6 +22,7 @@
 import shutil
 import threading
 import time
+from typing import List
```

Comment:
I think we can get rid of this and just use `list` directly now that we dropped Python 3.7 support.